spark-jobs/src/main/scala/za/co/absa/enceladus/common/plugin/menas/EventListenerMenas.scala

```diff
@@ -97,6 +117,7 @@ class EventListenerMenas(dao: MenasDAO,
       }
       try {
         dao.updateRunStatus(uniqueId, statusToSave)
+        notifyPlugins()
```

Comment:
How is this line of code covered by tests?

Reply:
We can implement a dummy plugin later so that this can be properly tested

Reply:
Agreed, the dummy can be even used to showcase the feature